substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -194,9 +212,13 @@ pub mod pallet {
 		/// The maximum size in bytes submitted evidence is allowed to be.
 		#[pallet::constant]
 		type EvidenceSize: Get<u32>;
+
+		#[pallet::constant]
+		type MaxRank: Get<u32>;
```

Comment:
Docs please. Also also a comment on that reducing this value can result in broken state.

Reply:
Done